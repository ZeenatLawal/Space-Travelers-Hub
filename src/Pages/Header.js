const Header = () => (
  <div className="headerWrap">
    <img src="../planet.png" alt="haha" className="logoImg" />
    <h1>Space Traveler&apos;s Hub</h1>
  </div>
);

export default Header;
