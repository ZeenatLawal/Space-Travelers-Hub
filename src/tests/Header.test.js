import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import Header from '../Pages/Header';

it('renders correctly', () => {
  const header = renderer
    .create(
      <Provider store={store}>
        <Header />
      </Provider>,
    )
    .toJSON();
  expect(header).toMatchSnapshot();
});
