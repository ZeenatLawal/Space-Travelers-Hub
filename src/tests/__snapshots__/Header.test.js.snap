// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="headerWrap"
>
  <img
    alt="planet"
    className="logoImg"
    src="space.png"
  />
  <h1>
    Space Traveler's Hub
  </h1>
</div>
`;
